### ISSUE TITLE
Integration: Agent Zero

### ISSUE THREAD

**Issue by bhupesh-sf** [av/harbor#23]
Hey this is quite cool project and quite active. Let me know if this will be feasible

[https://github.com/frdel/agent-zero](https://github.com/frdel/agent-zero)

**Comment by av** [comment]
Thanks for bringing it to my attention, it should be doable from the first glance, I'll be able to understand better once started with the integration

**Comment by bhupesh-sf** [comment]
yeah, agreed because that is by design. not sure what can be done in this case

**Comment by av** [comment]
Been a while!
Just revisited Agent Zero - they have proper docker support since v0.8.0, added to [Harbor v0.3.9](https://github.com/av/harbor/releases/tag/v0.3.9)

**Comment by bhupesh-sf** [comment]
Thanks a lot
